test/test_pipeline.cpp

```diff
@@ -21,6 +21,26 @@ namespace nvfuser {
 
 using namespace at::indexing;
 
+std::string sortByLine(const std::string& input) {
+  auto ss = std::stringstream(input);
+  std::vector<std::string> lines;
+  std::string line;
+  while (std::getline(ss, line, '\n')) {
+    lines.push_back(line);
+  }
+  std::sort(lines.begin(), lines.end());
+  std::stringstream output;
+  bool first = true;
+  for (auto line : lines) {
+    if (!first) {
+      output << std::endl;
+    }
+    first = false;
+    output << line;
+  }
+  return output.str();
+}
```

Comment:
Just cleaning up the TODO in this test while I'm here. This makes the error message provide a useful diff.